ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/ansible_eos_device.py

```diff
@@ -17,7 +17,13 @@
 
 logger = getLogger(__name__)
 
+REQUIRED_ANTA_VERSION = "v0.13.0"
+
 try:
+    from anta import __version__ as anta_version
+
+    if anta_version != REQUIRED_ANTA_VERSION:
+        raise AristaAvdError(message=f"AVD requires 'anta' Python library version v0.13.0, found {anta_version}")
```

Comment:
```suggestion
        raise AristaAvdError(message=f"AVD requires 'anta' Python library version {REQUIRED_ANTA_VERSION}, found {anta_version}")
```

Reply:
Fixed in 183de5840e99bdad7fdf14a5aea6e20aa8016765